src/sentry/static/sentry/app/views/projects/projectContext.jsx

```diff
@@ -95,6 +95,13 @@ const ProjectContext = createReactClass({
       this.fetchData();
     }
 
+    // Project list has changed. Likely indicating that a new project has been
+    // added. Refect project details in case that the new project is the active
+    // project.
+    if (prevProps.projects.length !== this.props.projects.length) {
```

Comment:
Can you add a comment here about why we're only comparing list sizes?

Reply:
yup.